plugins/package-managers/node/src/main/kotlin/NodePackageManagerType.kt

```diff
@@ -30,10 +30,10 @@ import org.ossreviewtoolkit.utils.common.collectMessages
  * An enum of all supported Node package managers.
  */
 enum class NodePackageManagerType(
-    val projectType: String,
-    val lockfileName: String,
-    val markerFileName: String? = null,
-    val workspaceFileName: String = NodePackageManagerType.DEFINITION_FILE
+    internal val projectType: String,
```

Comment:
I'm unsure about the benefit of this change, esp. as it increases maintenance effort because you have to remember to make any future additional property internal as well. 

Could you explain what's the advantage of making all properties `internal` while the enum itself needs to stay `public`?

Reply:
> Could you explain what's the advantage of making all properties `internal` while the enum itself needs to stay `public`?

minimize public API to not worry about impact of changes outside of ORT and remove the need to use `!` in any signature changes.